examples/v1beta1/trial-images/pytorch-mnist/Dockerfile

```diff
@@ -5,6 +5,7 @@ WORKDIR /opt/pytorch-mnist
 
 # Add folder for the logs.
 RUN mkdir /katib
+RUN pip install cloudml-hypertune
```

Comment:
Let's install specific version of `cloudml-hypertune`.

Reply:
It makes sense.

Reply:
Done.